### ISSUE TITLE
FM: Style is applied, but listed under "Other Scripts & CSS"

### ISSUE THREAD

**Issue by rubaboo** [erosman/support#282]
For example, if I navigate from [here](https://workflowy.com/s/demo-for-277/j8HnPbQ3XiTjZQln) to [there](https://workflowy.com/s/demo-for-277/j8HnPbQ3XiTjZQln#/2df487709b30), the style below is still applied (as expected, for sure), but is now listed under "Other Scripts & CSS" in the FM menu.
```
/*
==UserCSS==
@name             WF_Solarized
@match            https://workflowy.com/s/demo-for-277/j8HnPbQ3XiTjZQln
@version          1.0
==/UserCSS==
*/

#app {
  background-color: #eee8d5;
}
```

**Comment by erosman** [comment]
That is the peculiarity of the URLs but I will check and fix it.

This URL matches `@match`
`https://workflowy.com/s/demo-for-277/j8HnPbQ3XiTjZQln`

This URL is treated by Firefox as a hash of the first URL (and anchor on the same page, not a new page) and that is why `@match` matches.
`https://workflowy.com/s/demo-for-277/j8HnPbQ3XiTjZQln`**#/2df487709b30**

**Comment by erosman** [comment]
Fixed for v2.17
